client/pom.xml

```diff
@@ -478,6 +478,12 @@
                 <copy todir="${basedir}/target/generated-webapp/WEB-INF/classes/scripts">
                   <fileset dir="${basedir}/../scripts"/>
                 </copy>
+                <copy todir="${basedir}/target/generated-webapp/WEB-INF/classes/systemvm/js">
```

Comment:
Do we have any unit-tests added for this change? As well, if we possible, can we have a document for the console proxy workflow available on wiki ? This may help people to know the exact workflow or class interactions with business flow.  

As well, a small change to this request message "Made the adding new keyboard language support easier" could be beneficial i believe, helps people to know the complete context for this change. 

Note :this is not specific to this line, but a generic note.